unit_tests/testVTUFiles/testVTUFiles.f90

```diff
@@ -52,17 +51,23 @@ SUBROUTINE SetupError()
 !-------------------------------------------------------------------------------
     SUBROUTINE testClear()
       !Test clear
+      CALL testVTUFile%fopen()
+      testVTUFile%hasMesh=.TRUE.
+      !
       CALL testVTUFile%clear()
-      bool=.NOT.testVTUFile%isInit()
-      ASSERT(bool,'testVTUFile%clear(...)')
+      ASSERT(.NOT.testVTUFile%isInit(),'testVTUFile%isInit()')
+      ASSERT(.NOT.testVTUFile%isOpen(),'testVTUFile%isOpen()')
+      ASSERT(.NOT.testVTUFile%hasMesh,'testVTUFile%hasMesh')
     ENDSUBROUTINE testClear
 !
 !-------------------------------------------------------------------------------
     SUBROUTINE testInitialize()
       !Test initialize
       CALL testVTUFile%initialize(666,'testVTU0.vtu')
-      bool=testVTUFile%isInit().AND.testVTUFile%isOpen()
-      ASSERT(bool,'testVTUFile%initialize(...)')
+      ASSERT(testVTUFile%isInit(),'testVTUFile%isInit()')
+      ASSERT(testVTUFile%isOpen(),'testVTUFile%isOpen()')
+      ASSERT(.NOT.testVTUFile%hasMesh,'testVTUFile%hasMesh')
+      ASSERT(testVTUFile%numDataSet==0,'testVTUFile%numDataSet')
```

Comment:
More formatting stuff.
- I think its better to remove the "testVTUFile" from the message for assert. This part of the message is kind of redundant.
- Relational operators should have surrounding whitespace (e.g. `==` should be ` == `).